### PR TITLE
feat: adds attributes to ext apis #532

### PR DESCRIPTION
**Pull request by srinandan**

### CODE REVIEW

File: internal/client/hub/hub.go [hunk 1]
```diff
@@ -836,7 +836,7 @@ func ListExternalAPIs(filter string, pageSize int, pageToken string) (respBody [
 }
 
 func UpdateExternalAPI(externalApiID string, displayName string, description string,
-	endpoints []string, paths []string, externalUri string,
+	endpoints []string, paths []string, externalUri string, apiType string,
```
Comment (ssvaidyanathan) [inline internal/client/hub/hub.go:839]: shouldnt "apiType" arg be called "attributes"?

File: internal/client/hub/hub.go [hunk 2]
```diff
@@ -865,7 +865,7 @@ func UpdateExternalAPI(externalApiID string, displayName string, description str
 	q.Set("updateMask", strings.Join(updateMask, ","))
 	u.RawQuery = q.Encode()
 
-	payload, err := getExternalApi(displayName, description, endpoints, paths, externalUri)
+	payload, err := getExternalApi(displayName, description, endpoints, paths, externalUri, apiType)
```
Comment (ssvaidyanathan) [inline internal/client/hub/hub.go:868]: same comment as above

File: internal/client/hub/hub.go [hunk 3]
```diff
@@ -905,6 +906,19 @@ func getExternalApi(displayName string, description string,
 		e.Endpoints = endpoints
 	}
 
+	a, err := getAttributeAllowedlue(attribute, 0)
```
Comment (ssvaidyanathan) [inline internal/client/hub/hub.go:909]: did you mean to called the method "getAttributeAllowedValues" ?

File: internal/client/hub/hub.go (hunk 2 above)
Comment (srinandan) [reply internal/client/hub/hub.go:868]: No, this is correct. It is not attribute but `apiType`.

File: internal/client/hub/hub.go (hunk 1 above)
Comment (srinandan) [reply internal/client/hub/hub.go:839]: No, this is correct. It is not attribute but `apiType`.

File: internal/client/hub/hub.go (hunk 3 above)
Comment (srinandan) [reply internal/client/hub/hub.go]: Yes, fixing it.

**Review by ssvaidyanathan — APPROVED** [review]

**Pull request merged**